biz.aQute.bndlib/src/aQute/bnd/osgi/Jar.java

```diff
@@ -467,16 +640,26 @@ Optional<Manifest> manifest() {
 		}
 	}
 
-	Optional<ModuleAttribute> moduleAttribute() throws Exception {
```

Comment:
It looks like we lost a method here which would break version compatibility. It could just delegate to the new method with version `0`.

Reply:
Done